src/components/faq.tsx

```diff
@@ -84,3 +60,254 @@ const AccordianMenu = ({
     </div>
   );
 };
+
+export const faqs: FAQ[] = [
+  {
+    id: "what-mento",
+    question: "What is the Mento Protocol?",
+    answer: (
+      <span>
+        <Link
+          className="font-fg text-primary-blue underline text-sm"
+          href="https://www.mento.org/"
+          target="_blank"
+        >
+          Mento
+        </Link>{" "}
+        is an EVM-compatible protocol that enables creation of onchain digital
+        assets. The goal of the Platform is to onboard a billion Web3 users.
+      </span>
+    ),
+  },
+  {
+    id: "what-token",
+    question: "What will the MENTO token do?",
+    answer: (
+      <span>
+        The MENTO token is an ERC-20 token on the Celo Blockchain that will
+        grant access to a tailored governance structure that specifically caters
+        to the unique needs of the Mento Platform.
+      </span>
+    ),
+  },
+  {
+    id: "what-airdrop",
+    question: "What is an airdrop, and how will it be performed?",
+    answer: (
+      <>
+        <span>
+          An airdrop is the process of distributing MENTO governance tokens to
+          the existing community members as a reward for their past
+          contributions to the development and usage of Mento decentralized
+          stable assets or the Celo ecosystem overall.
+        </span>
+        <span>
+          Eligible Celo addresses can claim a combined max of 5% of the overall
+          total supply. Users claiming the airdrop will receive it as veMENTO
+          locked for 2 years (linearly unlocking without a cliff). The duration
+          of an airdrop campaign is 2 months. After the airdrop period ends,
+          unclaimed amounts will go to the Mento Community Fund.
+        </span>
+      </>
+    ),
+  },
+  {
+    id: "what-vemento",
+    question: "What is veMENTO?",
+    answer: (
+      <>
+        <span>
+          veMENTO (vote-escrowed MENTO) is a locked version of the MENTO token
+          that allows its holders to participate in the governance process of
+          the Mento Protocol. The veModel is an on-chain governance framework
+          pioneered by Curve Finance and adopted by multiple protocols in the
```

Comment:
Out of scope but could add reference links here

Reply:
Yeah, this is from @olenovyk's [notion page](https://www.notion.so/mentolabs/Slim-version-of-an-FAQ-for-airdrop-app-a0e7ad88dd8d43f996579545133ba8e7). I'll defer to him on the reference links